### DISCUSSION TITLE
GitHub README just wont render at repo homepage

### DISCUSSION THREAD

**Discussion by SaicharanKandukuri** [community/community#22420] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/1/010b5b00c2b4b15683d2d9487711a014b4465c74.png" data-download-href="/uploads/short-url/9eO05ybOSgwRy8YNzIH46Rzcyg.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/68287637/181070836-5422bcf3-9b2e-4017-b1a0-f2404cab9da7.png" alt="image" data-base62-sha1="9eO05ybOSgwRy8YNzIH46Rzcyg" width="690" height="137" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/1/010b5b00c2b4b15683d2d9487711a014b4465c74_2_690x137.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/1/010b5b00c2b4b15683d2d9487711a014b4465c74_2_1035x205.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/1/010b5b00c2b4b15683d2d9487711a014b4465c74.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/1/010b5b00c2b4b15683d2d9487711a014b4465c74_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1127×225 11.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>ok I made <code>README.md</code> it won’t render at repo home page</p>
<p>REPO URL: <a href="https://github.com/RandomCoderOrg/fs-cook" rel="noopener nofollow ugc">RandomCoderOrg/fs-cook: [DEV] Linux tarball building (github.com)</a></p>
<p>also tried names like</p>
<ul>
<li><code>readme.md</code></li>
<li><code>readme.txt</code></li>
<li>
<code>README.md</code><br>
but no luck</li>
</ul>
<p>I don’t know what to do!<br>
Help!</p>

**Comment by airtower-luna** [comment]
<p>Try capitalizing the file name (<code>README.md</code> or <code>README.txt</code>). Good luck! :slightly_smiling_face:</p>

**Comment by SaicharanKandukuri** [comment]
<p>actually, i tried that to <a href="https://github.com/RandomCoderOrg/fs-cook/commit/7186cb9cad9a89bc4d48fb320a73e903a89f2aeb" rel="noopener nofollow ugc">Rename README.md to readme.md · RandomCoderOrg/fs-cook@7186cb9 (github.com)</a></p>
<p>but still it wont render</p>

**Comment by airtower-luna** [comment]
<p>Huh, that looks buggy. :bug: I can only say capital README works in my repositories.</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/5/75063a4ce3534466c37856679214d9d750694dc6.png" data-download-href="/uploads/short-url/gHfcPlcvNYvID99lNdqg9jWloeG.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181070840-f241e0a7-bf55-4e27-a923-361ac8d44e12.png" alt="image" data-base62-sha1="gHfcPlcvNYvID99lNdqg9jWloeG" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/5/75063a4ce3534466c37856679214d9d750694dc6_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/5/75063a4ce3534466c37856679214d9d750694dc6_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/5/75063a4ce3534466c37856679214d9d750694dc6_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/5/75063a4ce3534466c37856679214d9d750694dc6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 240 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>it’s already embedded now, right? please check it again,</p>

**Comment by SaicharanKandukuri** [comment, marked as answer]
<p>Opps i forgot to close this<br>
I got help from GitHub support team</p>
